### PR TITLE
Add filters and search fields to Admins

### PR DESCRIPTION
**Pull request by estyxx**

### CODE REVIEW

File: backend/submissions/admin.py [hunk 1]
```diff
@@ -83,7 +83,14 @@ class SubmissionAdmin(AdminUsersMixin):
...
     )
     list_filter = ("conference", "type", "topic", "status")
-    search_fields = ("title", "abstract")
+    search_fields = (
+        "title",
+        "elevator_pitch",
+        "abstract",
+        "notes",
+        "previous_talk_video",
+        "comments__text",
```
Comment (marcoacierno) [inline backend/submissions/admin.py:92]: do we think it might be helpful to search the comments too?

Comment (estyxx) [reply backend/submissions/admin.py:92]: Maybe when you only remember "hey someone has commented to that pr about...."  ? 
I don't know tbh, I added anything that can be helpful but I should ask to the people that will use this admin...

Comment (estyxx) [reply backend/submissions/admin.py:92]: well I can remove it and we can add if it's needed...

**Review by marcoacierno — APPROVED** [review]

**Pull request merged**